.github/actions/publish-cypress-report/action.yml

```diff
@@ -0,0 +1,26 @@
+name: "publish-cypress-report"

```

Comment:
why not just creating a javascript action, instead of running your script using node bash script  ?

Reply:
Because I've started writting it in bash and I want to be sure that the report is published up to the GITHUB_STEP_SUMMARY